### DISCUSSION TITLE
Use the same package names as lspconfig

### DISCUSSION THREAD

**Discussion by raj-magesh** [mason-org/mason.nvim#241] [API suggestions]
I noticed this mapping table for packages (https://github.com/williamboman/mason-lspconfig.nvim/blob/main/doc/server-mapping.md) and wondered why there are two sets of package names. Is one any better than the other? It adds a (minor) extra level of confusion that could be avoided.

**Comment by williamboman** [comment]
The package names in Mason follow upstream naming conventions. An example of this would be the [`lua-language-server`](https://github.com/sumneko/lua-language-server) - it's not called `sumneko_lua` anywhere else than in `lspconfig`. `lspconfig` is also an optional plugin, you are very much able to use the LSP client without it (Mason itself doesn't care how you use installed packages). `mason-lspconfig` exists to help bridge these gaps, and I've also been thinking of including the `lspconfig` name of a package in the UI if the user has `mason-lspconfig` installed.

**Reply by raj-magesh** [reply]
I see, that makes a lot of sense, thanks for the explanation! I wonderwhy lspconfig chose to use non-upstream package names.
